platform/macos/src/MGLMapView.mm

```diff
@@ -2275,6 +2281,17 @@ - (void)updateAnnotationCallouts {
     return _mbglMap;
 }
 
+- (MGLStyle *)style
+{
+    if (_isWaitingForStyleLoad) {
```

Comment:
Add `&& _style` to silence the initial warning triggered by the attribution view. The style isn’t set by the time `-commonInit` calls `-installAttributionView`.